docker-compose.yml

```diff
@@ -156,7 +156,11 @@ services:
     build:
       context: docker/images/gcs-emulator
     image: local/tecken_gcs_emulator
-    command: -port 8001 -scheme http
+    command: >-
```

Comment:
Things I learned: the `>-` replaces newlines with spaces and chomps the last newline.

Reply:
I find it impossible to remember the various options for YAML multiline strings. The only thing I remember is that there is https://yaml-multiline.info/, and everytime I need something unsual I look it up there.